PinpointKit/PinpointKit/Sources/Fonts.swift

```diff
@@ -32,7 +32,7 @@ public extension UIFont {
      
      - returns: A Source Sans Pro `UIFont` at the specified size and weight.
      */
-    public static func sourceSansProFontOfSize(_ fontSize: CGFloat, weight: FontWeight = .regular) -> UIFont {
+    public static func sourceSansProFont(ofSize fontSize: CGFloat, weight: FontWeight = .regular) -> UIFont {
```

Comment:
> But I see how you could make an argument similar to UIColor.whateverColor() → UIColor.whatever().

This is exactly how I’m thinking of it. I think we can safely remove both, but if we want to be a bit more conservative just the first one.